pkg/api/frontendsettings.go

```diff
@@ -226,6 +226,7 @@ func (hs *HTTPServer) getFrontendSettingsMap(c *models.ReqContext) (map[string]i
 		},
 		"featureToggles":    hs.Cfg.FeatureToggles,
 		"rendererAvailable": hs.RenderService.IsAvailable(),
+		"http2Enabled":      setting.Protocol == "h2",
```

Comment:
Could you merge with master and use `hs.Cfg.Protocol == "h2"` here instead? We are trying to stop using `setting.*` global variables, and I just merged a PR to add `hs.Cfg.Protocol`.

Reply:
Of course :)